src/frontend/utils/toggles.ts

```diff
@@ -21,4 +21,5 @@ export enum Toggle {
      * features-under-utvikling
      */
     KAN_BRUKE_VEDTAKSPERIODER_TILSYN_BARN = 'sak.kan-bruke-vedtaksperioder-tilsyn-barn',
+    SKAL_BRUKE_MANED_AR_VELGER = 'sak.skal-bruke-maned-ar-velger',
```

Comment:
Ikke så viktig, men kan bruke 
```suggestion
    SKAL_BRUKE_MANED_AR_VELGER = 'sak.frontend.skal-bruke-maned-ar-velger',
```
hvis dette kun skal være et frontend flag